python/py-grpcio/Portfile

```diff
@@ -4,24 +4,22 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-grpcio
-version             1.25.0
+version             1.35.0
```

Comment:
please double-check that this latest version still supports PY27. If not, you'll need to restrict the py27 subport to the latest supported version.

Reply:
Based on the info at https://pypi.org/project/grpcio/, Python 2.7 appears to be supported with 1.35.0.

Tests also succeed:
```
$ sudo port test py27-grpcio
--->  Computing dependencies for py27-grpcio
--->  Fetching distfiles for py27-grpcio
--->  Verifying checksums for py27-grpcio
--->  Extracting py27-grpcio
--->  Configuring py27-grpcio
--->  Building py27-grpcio
--->  Testing py27-grpcio
$
```

Reply:
that's not how I read this: "Python >= 3.5" that's why I mentioned it ;) But I agree that this does not match with what they have in their `setup.py` and also with `pip` one installs version 1.35.0 for PY27; and if the tests pass I am fine with it.